matcher/service/openreview_interface.py

```diff
@@ -67,7 +91,9 @@ def papers(self):
                             key, value = element.split('.')[1].split('=')
                             content_dict[key] = value
                         else:
-                            self.logger.debug('Invalid filter provided in invitation: {}. Supported filter format "content.field_x=value1".'.format(element))
+                            error_msg = 'Invalid filter provided in invitation: {}. Supported filter format "content.field_x=value1".'.format(element)
+                            self.set_status('Error', error_msg)
```

Comment:
same here, set_status has to be called by the caller when it catches this exception.

Reply:
Fixed